specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/SessionPools_CreateOrUpdate.json

```diff
@@ -0,0 +1,150 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "sessionPoolName": "testsessionpool",
+    "api-version": "2024-02-02-preview",
+    "sessionPoolEnvelope": {
+      "location": "East US",
```

Comment:
Isn't it should `eastus` as recommended Azure region name ?
Please check implementation too.

Reply:
I tried again using az cli --debug, and in the debug message I think it is using similar pattern as "East US"

Reply:
I think East US should probably be fine, its accepted by most ARM APIs...